android/build.gradle

```diff
@@ -124,6 +124,7 @@ buildscript {
 
     ext.cmakeArgs = [
         "--no-warn-unused-cli",
+        "-DANDROID_WEAK_API_DEFS=ON",
```

Comment:
q: what is this for?  just wondering if we need to make the same change in our internal build file.